src/app/shared/components/input-currency/input-currency.component.ts

```diff
@@ -7,10 +7,27 @@ import { FormGroup } from '@angular/forms';
   styleUrls: ['./input-currency.component.scss'],
 })
 export class InputCurrencyComponent {
+  @Output() emitterValorVerificado: EventEmitter<boolean> =
+    new EventEmitter<boolean>();
+
   @Input() form!: FormGroup;
   @Input() type: string = 'text';
   @Input() class = 'form-control rounded-pill';
   @Input() id = '';
   @Input() frmName: string = '';
   @Input() placeholder = '';
+  @Input() valor: number = 0;
+
+  public valorValido: boolean = true;
+
+  verificaValor() {
+    let valorInput = this.form.controls[this.frmName].value;
+
```

Comment:
const valorValido = valorInput <= this.valor;
 this.emitterValorVerificado.emit(valorValido );